lib/MusicBrainz/Server/Entity/Statistics/ByDate.pm

```diff
@@ -47,22 +47,12 @@ __PACKAGE__->meta->make_immutable;
 no Moose;
 1;
 
-=head1 COPYRIGHT
+=head1 COPYRIGHT AND LICENSE
 
-Copyright (C) 2011 MetaBrainz Foundation Inc.
+Copyright (C) 2011 MetaBrainz Foundation
```

Comment:
Copyright line change (can revert if undesired, but we don't use `Inc.` elsewhere)